### PR TITLE
Fix/rhyme with diphthong

### LINKED ISSUE #108: Rhyme for hiatus with tilded vowels do not work properly
```
el Amor. Allá surge Sigurd que al Cid se aúna,
cerca de Dulcinea brilla el rayo de luna,
```
Endings are **AUNA** vs **UNA**

### PR DESCRIPTION
**Pull request by alvp**
Fixes #108 #104

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -112,6 +114,210 @@ Usage example
 Output example
 --------------
 
+
+The output of rantanplan is a complex structure that will be broke down for clarity.
```
Comment (versae) [inline README.rst]: `broken down`

File: README.rst [hunk 2]
```diff
@@ -112,6 +114,210 @@ Usage example
...
+
+First of all, rantanplan will show a list of stanzas. Each stanza is then showns as two separate lists. A list of tokens, and a list of "phonological groups" i.e., the phonological units that form a verse after synalephas and sinaereris are taken into acount.
```
Comment (versae) [inline README.rst]: Use just `First,` better. And capitalize `rantanplan`. Please, check for other typos (e.g, `showns`, `acount`)

File: src/rantanplan/rhymes.py [hunk 3]
```diff
@@ -49,7 +49,12 @@ def get_stressed_endings(lines):
         for phonological_group in line["phonological_groups"]:
             # Break groups on last synalepha index position if present
             if "synalepha_index" in phonological_group:
-                synalepha_index = phonological_group["synalepha_index"][-1] + 1
+                has_tilded_vowel = TILDED_VOWELS_RE.search(
+                    phonological_group["syllable"])
+                if has_tilded_vowel:
+                    synalepha_index = has_tilded_vowel.start()
+                else:
+                    synalepha_index = phonological_group["synalepha_index"][-1] + 1
```
Comment (versae) [inline src/rantanplan/rhymes.py]: This chunk could be refactor for both `synalepha_index` and `synaeresis_index`.

**Review by versae — CHANGES_REQUESTED** [review]
The new docs are superb! It could've been better in a separate PR though.

Also, a test is missing for the new code.

File: README.rst (hunk 2 above)
Comment (alvp) [reply README.rst]: Fixed all typos I found.

File: README.rst (hunk 1 above)
Comment (alvp) [reply README.rst]: Fixed

File: src/rantanplan/rhymes.py (hunk 3 above)
Comment (alvp) [reply src/rantanplan/rhymes.py]: Done by creating a new function and its tests

File: src/rantanplan/rhymes.py [hunk 4]
```diff
@@ -40,32 +40,34 @@
...
+    has_tilded_vowel = TILDED_VOWELS_RE.search(
+        phonological_group["syllable"])
+    if has_tilded_vowel:
+        liaison_index = has_tilded_vowel.start()
+    else:
+        liaison_index = phonological_group[f"{liaison}_index"][-1] + 1
+    syllable = phonological_group["syllable"][liaison_index:]
+    return syllable
+
+
```
Comment (versae) [inline src/rantanplan/rhymes.py:56]: Beautiful!

**Review by versae — APPROVED** [review]

**Pull request merged**